core/src/it/units/sdm/project/game/gui/screens/GameScreen.java

```diff
@@ -36,7 +37,7 @@ public GameScreen(@NotNull FreedomGame game) {
         stage = new Stage(new FitViewport(GAME_SCREEN_WORLD_WIDTH, GAME_SCREEN_WORLD_HEIGHT), new SpriteBatch());
         VisTable container = new VisTable();
         logArea = new VisTextArea("Welcome to Freedom ");
-        logArea.setPrefRows(game.getNumberOfRowsAndColumns() * 2);
+        logArea.setPrefRows(game.getNumberOfRowsAndColumns() * 2.0f);
```

Comment:
Is this a SonarQube suggestion? Also, 2f might be sufficient

Reply:
yes, you are right

Reply:
I reverted back to the version with no casting then

Reply:
I have already refactored that